biggraphite/drivers/cassandra.py

```diff
@@ -1994,18 +2017,28 @@ def clean(self, max_age=None, start_key=None, end_key=None, shard=1, nshards=0,
         """
         super(_CassandraAccessor, self).clean(max_age, callback_on_progress)
 
-        self._clean_empty_dir(start_key, end_key, shard, nshards, callback_on_progress)
+        #self._clean_empty_dir(start_key, end_key, shard, nshards, callback_on_progress)
```

Comment:
why comment that line? if it is not needed delete it

Reply:
typo